Uni_life_tracker/app/src/main/java/com/generic/ult/Expenses_list.java

```diff
@@ -0,0 +1,14 @@
+package com.generic.ult;
+
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+
+public class Expenses_list extends AppCompatActivity {
```

Comment:
I think the name of the class should ExpensesList with no underscore

Reply:
Fixed